symfony/security-bundle/7.3/config/packages/security.yaml

```diff
@@ -2,36 +2,39 @@ security:
     # https://symfony.com/doc/current/security.html#registering-the-user-hashing-passwords
     password_hashers:
         Symfony\Component\Security\Core\User\PasswordAuthenticatedUserInterface: 'auto'
+
     # https://symfony.com/doc/current/security.html#loading-the-user-the-user-provider
     providers:
         users_in_memory: { memory: null }
+
     firewalls:
+        # Disable security for dev tools and static assets (only needed if access_control is broad):
         dev:
-            pattern: ^/(_(profiler|wdt)|css|images|js)/
+            # 'assets/' is for AssetMapper, 'build/' for Webpack Encore.
+            # (Note: no regex delimiters needed; Symfony adds `{}` automatically.)
```

Comment:
```suggestion
        dev:
            # Ensure dev tools and static assets are always allowed
```

Reply:
>there's no need to escape those slashes.

 yet if one does it still works, isn't it?
then we don't need to tell - config files shouldn't be documentation

Reply:
> Indeed, but there is no documentation for this - so there is no better place ;-)

The better place is contributing to the documentation so that there is documentation for this.